tests/test_aiocassandra.py

```diff
@@ -178,9 +184,13 @@ def new_patch_factory():
 
     paginator = cassandra.execute_futures(statement)
 
+    ret = []
+
     with pytest.raises(RuntimeError):
-        async for _ in paginator:
-            _
+        async for row in paginator:
+            ret.appen(row)
```

Comment:
append

Reply:
good catch

Reply:
Fixed by https://github.com/aio-libs/aiocassandra/commit/5890944d2aceb8a4f89566677d739bbcafa34a3a

Thanks for the report!

Reply:
You are welcome )